app/views/groups/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<p style="color: green"><%= notice %></p>
+<br/>
+<%= render @group %>
+
+<div class="footer-buttons">
+  <%= link_to "Back to groups", groups_path, class:"inner-buttons" %>
+
+  <%= button_to "Destroy this group", @group, method: :delete, class:"new-buttons" %>
+</div>

```

Comment:
- [ ] just as the previous reviewer mentioned, this page should show a list of transactions made for this particular group and also a button to add more transactions.
your current design
![Screenshot from 2023-08-12 00-46-55](https://github.com/joyapisi/budget-app/assets/66097371/d7ec6099-66fc-45e4-ae04-6d288387e42a)